app/lib/katello/resources/candlepin/upstream_consumer.rb

```diff
@@ -42,19 +42,24 @@ def remove_entitlement(entitlement_id)
 
           def get_export(url, client_cert, client_key, ca_file)
             logger.debug "Sending GET request to upstream Candlepin: #{url}"
-            return resource(url: url, client_cert: client_cert, client_key: client_key, ca_file: ca_file).get
+            resource(url: url, client_cert: client_cert, client_key: client_key, ca_file: ca_file).get
           rescue RestClient::Exception => e
             raise e
           end
 
           def update(url, client_cert, client_key, ca_file, attributes)
             logger.debug "Sending PUT request to upstream Candlepin: #{url} #{attributes.to_json}"
-            return resource(url: url, client_cert: client_cert, client_key: client_key, ca_file: ca_file).put(attributes.to_json,
+            resource(url: url, client_cert: client_cert, client_key: client_key, ca_file: ca_file).put(attributes.to_json,
```

Comment:
Any way we can break this up? 123 characters on this line and some wild formatting on the next few lines.

Reply:
let's see what Rubocop will let me do :)

We do have a pretty generous max line length in our config for some reason. :man_shrugging: